QuizCheckResult.java

```diff
@@ -0,0 +1,147 @@
+import java.util.*;
+class QuizCheckResult
+{
+    public static final int PASSWORD_LENGTH = 8;
+
+    public static boolean is_Valid_Password(String password) {
```

Comment:
Use camelCase for method names instead of snake_case

Reply:
done. please check once .